WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -52,6 +52,7 @@ + (NSArray *)routes
     [[FBRoute POST:@"/wda/getPasteboard"] respondWithTarget:self action:@selector(handleGetPasteboard:)],
     [[FBRoute GET:@"/wda/batteryInfo"] respondWithTarget:self action:@selector(handleGetBatteryInfo:)],
     [[FBRoute POST:@"/wda/pressButton"] respondWithTarget:self action:@selector(handlePressButtonCommand:)],
+    [[FBRoute POST:@"/wda/siri/activate"] respondWithTarget:self action:@selector(handleActivateSiri:)],
```

Comment:
is `activate` a good term? To me "activate" makes it sound like we are enabling siri or turning it on, but not yet giving it a command to process. what about `/wda/siri/voice_command` and `handleSiriVoiceCommand` or something similar?

Reply:
cool. what i care most about will be the user facing mobile method name, so let's talk about it more there. fine to merge this as is.